projects/playground/src/app/app.component.ts

```diff
@@ -53,8 +50,9 @@ export class AppComponent {
     const options: LogoutOptions = {
       localOnly: formOptions.localOnly === true,
       federated: formOptions.federated === true,
-      returnTo: this.window.location.origin,
+      returnTo: this.doc.location.origin,
```

Comment:
How does this "doc" behave vs the "window" object in terms of Angular SSR?

Reply:
It behaves better as I believe Angular itself can substitute this with something that works in SSR. However, it doesn't itself fix SSR support as there are other things within the underlying SDK that are preventing SSR from being supported fully.